excercise-e-shop/assets/scss/modules/_button.scss

```diff
@@ -0,0 +1,35 @@
+.btn {
+  padding: 14px 30px;
+  font-weight: $bold-weight;
+  background-color: #fff;
+  color: $base-text-color;
+  &.btn-primary {
+   padding: 14px 18px;
+   border-radius: 24px;
+  }
+  &.btn-secondary {
+    border-radius: 39px;
+    border: 1px solid #C4C4C4;
+  }
+  &.btn-subscribe {
+    border-radius: 56px;
+    background-color: $base-color;
+  }
+}
```

Comment:
I have said about modularization in this case, should using `%extend` in order to detailization from the base `class`